.github/workflows/update_merge_base_info.yml

```diff
@@ -0,0 +1,57 @@
+name: Update test file ratings for TD experiment
+
+on:
+  pull_request:
+  schedule:
+    - cron: 5 2 * * 5  # At 02:05 on Friday UTC or about 7pm PT Thursday
```

Comment:
We'll prob want to change this to run every night soon.  Any objections to giving it that cadence from the get-go?